pin.go

```diff
@@ -58,7 +95,7 @@ func createPinNVIndex(tpm *tpm2.TPMContext, handle tpm2.Handle, updateKeyName tp
 
 	nameAlg := tpm2.HashAlgorithmSHA256
 
-	// The NV index requires 4 policies:
+	// The NV index requires 5 policies:
 	// - A signed policy for initialization, signed with a key that is discarded so that the index cannot be recreated.
 	// - A signed policy for updating the index to revoke old dynamic authorization policies.
 	// - A policy for updating the authorization value (PIN / passphrase).
```

Comment:
maybe the code below maybe should mark each section with a comment again repeating which policy it is about

Reply:
I've updated the comments and added comments to each subsequent block too.